ripple_api/ripple_api.py

```diff
@@ -153,6 +153,33 @@ def call_api(data, servers=None, server_url=None, api_user=None,
     raise error
 
 
+def account_info(account, servers=None, server_url=None, api_user=None,
+                 api_password=None, timeout=5):
+    """ Fetches from rippled Information about account
+
+        params:
+            account -- ripple-address of account to be detailed
+            servers, server_url, api_user, api_password, time -- see call_api
+        returns:
+            dictionary with account info. See:
+            https://ripple.com/build/rippled-apis/#account-info
+    """
+
+    request = {
+        "method": "account_info",
+        "params": [
+            {
+                "account": account,
+                "strict": True,
+                "ledger_index": "validated"
+            }
+        ]
+    }
+
+    return call_api(request, servers=None, server_url=None, api_user=None,
```

Comment:
Why servers=None, server_url=None, etc, etc?  Why not use ones, that are submitted as parameters?